x-pack/plugins/security_solution/cypress/integration/timeline_creation.spec.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { timeline } from '../objects/timeline';
+
+import { TIMELINES } from '../screens/security_header';
+import {
+  FAVORITE_TIMELINE,
+  LOCKED_ICON,
+  NOTES,
+  NOTES_BUTTON,
+  NOTES_COUNT,
+  NOTES_TEXT_AREA,
+  PIN_EVENT,
+  TIMELINE_DESCRIPTION,
+  // TIMELINE_FILTER,
```

Comment:
nit: this may be deleted

Reply:
I know that it should be deleted but I left it on purpose, as soon as the bug with the filter is fixed, the 2 commented lines should be uncommented ;)